src/main/java/com/marcusslover/plus/lib/sound/Music.java

```diff
@@ -220,4 +193,55 @@ public void stopTasks() {
         this.tail = tail;
         return this;
     }
+
+    public Music play() {
+        this.sessions().forEach((audience, session) -> session.start());
+        return this;
+    }
+
+
+    private static class Session {
+        @Getter private final long startTime;
```

Comment:
You may actually put `@Getter` at the top of your class instead of adding it to each field, it's neater and cleaner to read.